src/store/application_settings.js

```diff
@@ -0,0 +1,34 @@
+
+export default {
+  namespaced: true,
+  state: {
+    current_reporting_period_id: null,
+    title: null
+  },
+  mutations: {
+    setApplicationSettings (state, applicationSettings) {
+      Object.assign(state, applicationSettings)
+    }
+  },
+  actions: {
+    reloadApplicationSettings ({ commit }) {
+      return fetch('/api/application_settings')
```

Comment:
Note to self: need to add `{credentials: 'include'}`

Reply:
> Note to self: need to add `{credentials: 'include'}`

i don't think this is actually true; i've been omitting it and things work fine, and i'm not really sure why it was added in the first place. we're not making cross-origin requests, and i *think* browsers default to sending them to same-origin sites (e.g., that the default is `same-origin`)

Reply:
Ah. I think you're right as long as these requests are all relative on the same host. In GOST dev mode, API_DOMAIN and WEBSITE_DOMAIN differ and the client served on localhost:8080 makes AJAX requests to localhost:3000, which I think is considered cross-origin? So that's probably where this came from originally (or rather, some common ancestor that had this behavior)

It seems annoying that GOST does that though and it would be easy to make everything be on the same origin in dev by setting
```
proxy: {
  '/api': {
    target: 'http://localhost:3000'
  }
}
```
in GOST's vue.config.js like ARPA Reporter has.